public/js/pimcore/object/helpers/gridTabAbstract.js

```diff
@@ -231,6 +231,45 @@ pimcore.object.helpers.gridTabAbstract = Class.create({
 
             this.store.getProxy().setExtraParam("query", this.searchFilter);
 
+            var selectObjectOptions = Ext.create('Ext.data.Store', {
+                fields: ['name', 'value'],
+                data: [
+                    [t("empty"), "all_objects"],
```

Comment:
```suggestion
                    [t("all_types"), "all_objects"],
```
recommending to use this existing translation key that sounds slightly better than `empty`, at least in this context